sandboxed_containers/deploying-sandboxed-container-workloads.adoc

```diff
@@ -0,0 +1,16 @@
+[id="deploying-sandboxed-containers-workloads"]
+= Deploying OpenShift sandboxed containers workloads
```

Comment:
You can use your `OpenShift sandboxed containers` attribute here

Reply:
PV2/FCC Compliance - not to use attributes in the title